index.html

```diff
@@ -143,8 +143,7 @@ <h3 class="center">Our work reflected in numbers</h3>
       </li>
       <li class="section center">
         <h4>Number of people helped out</h4>
-        <p>A lot
-          <3</p>
+        <p>A lot &lt</p>
```

Comment:
Great suggestion! This render wrong when I test locally however, resulting in only a '<' displayed (was expecting '<3' or heart symbol). How about replacing it with HTML code &#10084; for heart instead?